app/index.ejs

```diff
@@ -8,6 +8,27 @@
       name="description"
       content="The Event Registration Tool provides everything you need when it comes to event registration and management."
     />
+    <meta
+      property="og:url"
+      content="https://www.eventregistrationtool.com/"
+    />
+    <meta property="og:type" content="website" />
+    <meta
+      property="og:title"
+      content="Event Registration Tool"
+    />
+    <meta
+      property="og:image"
+      content="https://eventregistrationtool.com/assets/favicon.ico"
+    />
+    <meta
+      name="og:description"
+      content="The Event Registration Tool provides everything you need when it comes to event registration and management."
```

Comment:
I wonder if this should be a more registrant friendly message. I assume the most common way snippets would be shown is when people share registration links. This doesn't quite sound like you're clicking the link in order to register.

Reply:
I came up with this a while back when I saw the javascript code is set as the description when it is displayed as a search result on google. I asked Mike Schmitt for a better description a while ago but he never got back to me.

Reply:
@OzzieOrca Do you think this is good as is? We could always update it once Mike Schmitt gets back to me with an updated version of the description or if you have any suggestions on better verbiage.

Reply:
It's not a regression so feel free to merge it :)